### PR TITLE
[Keyboard] added Time 80 Reforged by Fox Lab

### PR DESCRIPTION
**Pull request by matrixzj**
* added Time 80 Reforged by Fox Lab
* split to two sub directories for universal and hotswap pcb
* Apply suggestions from code review

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/foxlab/time_re/hotswap/hotswap.c [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+void led_set_kb(uint8_t usb_led) {
+  if (IS_LED_ON(usb_led, USB_LED_CAPS_LOCK)) {
+    writePinLow(E6);
+  } else {
+    writePinHigh(E6);
+  }
+  led_set_user(usb_led);
+}
```
Comment (drashna) [inline keyboards/foxlab/time_re/hotswap/hotswap.c:39]:
```suggestion
```

File: keyboards/foxlab/time_re/hotswap/config.h [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ */
+#define MATRIX_ROW_PINS { D1, D0, B3, B0, B2, B1 }
+#define MATRIX_COL_PINS { F0, F1, F4, F5, F6, F7, C7, C6, B6, B5, B4, D7, D4, D6, D2, D3, D5 }
+
+#define DIODE_DIRECTION COL2ROW
+
+#define BACKLIGHT_PIN B7
+#define BACKLIGHT_LEVELS 3
+
+#define RGB_DI_PIN E2
```
Comment (drashna) [inline keyboards/foxlab/time_re/hotswap/config.h:52]:
```suggestion
#define LED_CAPS_LOCK_PIN E6
#define LED_PIN_ON_STATE 0

#define RGB_DI_PIN E2
```

File: keyboards/foxlab/time_re/universal/config.h [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+ */
+#define MATRIX_ROW_PINS { D1, D0, B3, B0, B2, B1 }
+#define MATRIX_COL_PINS { F0, F1, F4, F5, F6, F7, C7, C6, B6, B5, B4, D7, D4, D6, D2, D3, D5 }
+
+#define DIODE_DIRECTION COL2ROW
+
+#define BACKLIGHT_PIN B7
+#define BACKLIGHT_LEVELS 3
+
+#define RGB_DI_PIN E2
```
Comment (drashna) [inline keyboards/foxlab/time_re/universal/config.h:52]:
```suggestion
#define LED_CAPS_LOCK_PIN E6
#define LED_PIN_ON_STATE 0

#define RGB_DI_PIN E2
```

File: keyboards/foxlab/time_re/universal/universal.c [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+void led_set_kb(uint8_t usb_led) {
+  if (IS_LED_ON(usb_led, USB_LED_CAPS_LOCK)) {
+    writePinLow(E6);
+  } else {
+    writePinHigh(E6);
+  }
+  led_set_user(usb_led);
+}
```
Comment (drashna) [inline keyboards/foxlab/time_re/universal/universal.c:39]:
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by matrixzj** [comment]
@drashna  Please review , I have modified as suggested.

File: keyboards/foxlab/time_re/hotswap/rules.mk [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+SLEEP_LED_ENABLE = no		# Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no			# USB Nkey Rollover
+BACKLIGHT_ENABLE = no		# Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no		# Enable keyboard RGB underglow
+MIDI_ENABLE = no			# MIDI support
+UNICODE_ENABLE = no			# Unicode
+BLUETOOTH_ENABLE = no		# Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no			# Audio output on port C6
+FAUXCLICKY_ENABLE = no		# Use buzzer to emulate clicky switches
```
Comment (drashna) [inline keyboards/foxlab/time_re/hotswap/rules.mk:25]:
```suggestion
BLUETOOTH_ENABLE = no		# Enable Bluetooth
AUDIO_ENABLE = no			# Audio output
```

File: keyboards/foxlab/time_re/universal/rules.mk [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+SLEEP_LED_ENABLE = no		# Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no			# USB Nkey Rollover
+BACKLIGHT_ENABLE = no		# Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no		# Enable keyboard RGB underglow
+MIDI_ENABLE = no			# MIDI support
+UNICODE_ENABLE = no			# Unicode
+BLUETOOTH_ENABLE = no		# Enable Bluetooth with the Adafruit EZ-Key HID
+AUDIO_ENABLE = no			# Audio output on port C6
+FAUXCLICKY_ENABLE = no		# Use buzzer to emulate clicky switches
```
Comment (drashna) [inline keyboards/foxlab/time_re/universal/rules.mk:25]:
```suggestion
BLUETOOTH_ENABLE = no		# Enable Bluetooth
AUDIO_ENABLE = no			# Audio output
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by matrixzj** [comment]
Updated as suggested, please review @drashna

File: keyboards/foxlab/time_re/hotswap/config.h [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x464C // "FL"
+#define PRODUCT_ID      0x0006
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Fox Lab
+#define PRODUCT         Time 80 Reforged
+#define DESCRIPTION     Fox Lab Time 80 Reforged Hotswap
```
Comment (drashna) [inline keyboards/foxlab/time_re/hotswap/config.h:28]:
```suggestion
```

File: keyboards/foxlab/time_re/universal/config.h [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x464C // "FL"
+#define PRODUCT_ID      0x0006
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Fox Lab
+#define PRODUCT         Time 80 Reforged
+#define DESCRIPTION     Fox Lab Time 80 Reforged
```
Comment (drashna) [inline keyboards/foxlab/time_re/universal/config.h:28]:
```suggestion
```

File: keyboards/foxlab/time_re/hotswap/keymaps/via/keymap.c [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT(
+        KC_ESC,             KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,             KC_PSCR,  KC_SLCK,  KC_PAUS,
+        KC_GRV,   KC_1,     KC_2,     KC_3,     KC_4,     KC_5,     KC_6,     KC_7,     KC_8,     KC_9,     KC_0,     KC_MINS,  KC_EQL,   KC_BSPC,  _______,  KC_INS,   KC_HOME,  KC_PGUP,
+        KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,     KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_LBRC,  KC_RBRC,  KC_BSLS,            KC_DEL,   KC_END,   KC_PGDN,
+        KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,     KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_QUOT,            KC_ENT,
+        KC_LSFT,  KC_SPC,   KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_RSFT,  _______,                      KC_UP,
+        KC_LCTL,  KC_LGUI,  KC_LALT,                                KC_SPC,                                 MO(1),    _______,  KC_RGUI,  KC_RCTL,            KC_LEFT,  KC_DOWN,  KC_RGHT
+    ),
+    [1] = LAYOUT(
```
Comment (drashna) [inline keyboards/foxlab/time_re/hotswap/keymaps/via/keymap.c:27]: All 4 layers need to be populated for VIA, or the unfilled layers will have jibberish in them.

File: keyboards/foxlab/time_re/universal/keymaps/via/keymap.c [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```
Comment (drashna) [inline keyboards/foxlab/time_re/universal/keymaps/via/keymap.c:18]: needs 4 layers, as well.

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by matrixzj** [comment]
@drashna  keymap updated for 4 layers and add some keycodes for rgb logo light control. please review.

File: keyboards/foxlab/time_re/hotswap/keymaps/via/keymap.c [hunk 11]
```diff
@@ -0,0 +1,52 @@
...
+    ),
+    [2] = LAYOUT(
+        _______,            _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,            _______,  _______,  _______,
+        _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,
+        _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,            _______,  _______,  _______,
+        _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,            _______,
+        _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,  _______,                      _______,
+        _______,  _______,  _______,                                _______,                                _______,  _______,  _______,  _______,            _______,  _______, _______
+    ),
+    [1] = LAYOUT(
```
Comment (drashna) [inline keyboards/foxlab/time_re/hotswap/keymaps/via/keymap.c:43]:
```suggestion
    [3] = LAYOUT(
```

**Review by drashna — CHANGES_REQUESTED** [review]
Just one minor thing.

File: keyboards/foxlab/time_re/hotswap/keymaps/via/keymap.c (hunk 11 above)
Comment (matrixzj) [reply keyboards/foxlab/time_re/hotswap/keymaps/via/keymap.c]: fixed. sorry for oversight.

**Review by drashna — APPROVED** [review]

**Comment by matrixzj** [comment]
Is there anything I can improve? It has been pending for a while , just to make sure nothing is pending from my side.

File: keyboards/foxlab/time_re/hotswap/keymaps/default/keymap.c [hunk 12]
```diff
@@ -0,0 +1,36 @@
+/* Copyright 2020 REPLACE_WITH_YOUR_NAME
```
Comment (tzarc) [inline keyboards/foxlab/time_re/hotswap/keymaps/default/keymap.c:1]: Wrong name for copyright.

File: keyboards/foxlab/time_re/hotswap/keymaps/via/keymap.c [hunk 13]
```diff
@@ -0,0 +1,52 @@
+/* Copyright 2020 REPLACE_WITH_YOUR_NAME
```
Comment (tzarc) [inline keyboards/foxlab/time_re/hotswap/keymaps/via/keymap.c:1]: Wrong name for copyright.

File: keyboards/foxlab/time_re/universal/keymaps/default/keymap.c [hunk 14]
```diff
@@ -0,0 +1,36 @@
+/* Copyright 2020 REPLACE_WITH_YOUR_NAME
```
Comment (tzarc) [inline keyboards/foxlab/time_re/universal/keymaps/default/keymap.c:1]: Wrong name for copyright.

File: keyboards/foxlab/time_re/universal/keymaps/via/keymap.c [hunk 15]
```diff
@@ -0,0 +1,52 @@
+/* Copyright 2020 REPLACE_WITH_YOUR_NAME
```
Comment (tzarc) [inline keyboards/foxlab/time_re/universal/keymaps/via/keymap.c:1]: Wrong name for copyright.

**Review by tzarc — CHANGES_REQUESTED** [review]

File: keyboards/foxlab/time_re/universal/keymaps/via/keymap.c (hunk 15 above)
Comment (matrixzj) [reply keyboards/foxlab/time_re/universal/keymaps/via/keymap.c]: fixed. sorry for oversight.

File: keyboards/foxlab/time_re/universal/keymaps/default/keymap.c (hunk 14 above)
Comment (matrixzj) [reply keyboards/foxlab/time_re/universal/keymaps/default/keymap.c]: fixed. sorry for oversight.

File: keyboards/foxlab/time_re/hotswap/keymaps/via/keymap.c (hunk 13 above)
Comment (matrixzj) [reply keyboards/foxlab/time_re/hotswap/keymaps/via/keymap.c]: fixed

File: keyboards/foxlab/time_re/hotswap/keymaps/default/keymap.c (hunk 12 above)
Comment (matrixzj) [reply keyboards/foxlab/time_re/hotswap/keymaps/default/keymap.c]: fixed

**Comment by matrixzj** [comment]
Anything wrong with current pull ? still pending @tzarc

**Comment by matrixzj** [comment]
All above mentioned have been changed
`$ grep Copy keyboards/foxlab/time_re/hotswap/keymaps/default/keymap.c
/* Copyright 2021 matrixzj <matrix.zj@gmail.com>

$ grep Copy keyboards/foxlab/time_re/hotswap/keymaps/via/keymap.c
/* Copyright 2021 matrixzj <matrix.zj@gmail.com>

$ grep Copy keyboards/foxlab/time_re/universal/keymaps/default/keymap.c
/* Copyright 2021 matrixzj <matrix.zj@gmail.com>

$ grep Copy keyboards/foxlab/time_re/universal/keymaps/via/keymap.c
/* Copyright 2021 matrixzj <matrix.zj@gmail.com>
`

@tzarc  anything else changes will be needed?

**Comment by matrixzj** [comment]
@tzarc  please check what changes are still needed.

File: keyboards/foxlab/time_re/hotswap/info.json [hunk 16]
```diff
@@ -0,0 +1,104 @@
+{
+    "keyboard_name": "Time RE Hotswap",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 18.25,
+    "height": 6.5,
+    "layouts": {
+        "LAYOUT": {
+            "key_count": 90,
```
Comment (fauxpark) [inline keyboards/foxlab/time_re/hotswap/info.json:9]:
```suggestion
```

File: keyboards/foxlab/time_re/hotswap/readme.md [hunk 17]
```diff
@@ -0,0 +1,19 @@
+# Foxlab Time 80 Reforged Universal
+
+![Time 80 Reforged](https://i.imgur.com/BXOM9sQ.jpg)
```
Comment (fauxpark) [inline keyboards/foxlab/time_re/hotswap/readme.md:3]:
```suggestion
![Time 80 Reforged](https://i.imgur.com/BXOM9sQh.jpg)
```

File: keyboards/foxlab/time_re/hotswap/rules.mk [hunk 18]
```diff
@@ -0,0 +1,24 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no		# Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no			# USB Nkey Rollover
+BACKLIGHT_ENABLE = yes		# Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes		# Enable keyboard RGB underglow
+MIDI_ENABLE = no			# MIDI support
+UNICODE_ENABLE = no			# Unicode
+BLUETOOTH_ENABLE = no		# Enable Bluetooth
+AUDIO_ENABLE = no			# Audio output
```
Comment (fauxpark) [inline keyboards/foxlab/time_re/hotswap/rules.mk:24]:
```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/foxlab/time_re/universal/info.json [hunk 19]
```diff
@@ -0,0 +1,104 @@
+{
+    "keyboard_name": "Time Re Univeral",
+    "url": "",
+    "maintainer": "qmk",
+    "width": 18.25,
+    "height": 6.5,
+    "layouts": {
+        "LAYOUT": {
+            "key_count": 90,
```
Comment (fauxpark) [inline keyboards/foxlab/time_re/universal/info.json:9]:
```suggestion
```

File: keyboards/foxlab/time_re/universal/readme.md [hunk 20]
```diff
@@ -0,0 +1,20 @@
+# Foxlab Time 80 Reforged Universal
+
+![Time 80 Reforged](https://i.imgur.com/BXOM9sQ.jpg)
```
Comment (fauxpark) [inline keyboards/foxlab/time_re/universal/readme.md:3]:
```suggestion
![Time 80 Reforged](https://i.imgur.com/BXOM9sQh.jpg)
```

File: keyboards/foxlab/time_re/universal/readme.md [hunk 21]
```diff
@@ -0,0 +1,20 @@
...
+
+This version is for the universal variant with the layouts below available:
+![PCB Layout](https://imgur.com/D8cQCwb.png)
+![Plate Layout](https://imgur.com/RzeVma5.png)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make foxlab/time_re/universal:default
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start     with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
Comment (fauxpark) [inline keyboards/foxlab/time_re/universal/readme.md:19]:
```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```

File: keyboards/foxlab/time_re/universal/rules.mk [hunk 22]
```diff
@@ -0,0 +1,24 @@
...
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no		# Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no			# USB Nkey Rollover
+BACKLIGHT_ENABLE = yes		# Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = yes		# Enable keyboard RGB underglow
+MIDI_ENABLE = no			# MIDI support
+UNICODE_ENABLE = no			# Unicode
+BLUETOOTH_ENABLE = no		# Enable Bluetooth
+AUDIO_ENABLE = no			# Audio output
```
Comment (fauxpark) [inline keyboards/foxlab/time_re/universal/rules.mk:24]:
```suggestion
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = no            # USB Nkey Rollover
BACKLIGHT_ENABLE = yes      # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/foxlab/time_re/universal/rules.mk (hunk 22 above)
Comment (matrixzj) [reply keyboards/foxlab/time_re/universal/rules.mk]: Thanks for your suggestions @fauxpark  All has been updated, please review.

**Comment by matrixzj** [comment]
@tzarc  Please check

**Review by fauxpark — APPROVED** [review]

**Comment by matrixzj** [comment]
@tzarc is there any changes will be needed? please let me know.

File: keyboards/foxlab/time_re/hotswap/readme.md [hunk 23]
```diff
@@ -0,0 +1,19 @@
+# Foxlab Time 80 Reforged Universal
```
Comment (noroadsleft) [inline keyboards/foxlab/time_re/hotswap/readme.md:1]:
```suggestion
# Foxlab Time 80 Reforged Hotswap
```

File: keyboards/foxlab/time_re/hotswap/readme.md [hunk 24]
```diff
@@ -0,0 +1,19 @@
...
+
+![Time 80 Reforged](https://i.imgur.com/BXOM9sQh.jpg)
+
+Time 80 Reforged was the second design of Fox Lab back in 2017. It has also become an iconic keyboard design of Fox Lab. Time80RE kept most of the original weight design. The clock has become thinner now and h    as a tempered glass cover. The front height of the keyboard has greatly reduced for optimizing for comfort. The ratio between the top and bottom cases has been adjusted for a cleaner side view.
```
Comment (noroadsleft) [inline keyboards/foxlab/time_re/hotswap/readme.md:5]:
```suggestion
Time 80 Reforged was the second design of Fox Lab back in 2017. It has also become an iconic keyboard design of Fox Lab. Time80RE kept most of the original weight design. The clock has become thinner now and has a tempered glass cover. The front height of the keyboard has been greatly reduced for optimizing for comfort. The ratio between the top and bottom cases has been adjusted for a cleaner side view.
```

File: keyboards/foxlab/time_re/universal/info.json [hunk 25]
```diff
@@ -0,0 +1,103 @@
+{
+    "keyboard_name": "Time Re Univeral",
```
Comment (noroadsleft) [inline keyboards/foxlab/time_re/universal/info.json:2]:
```suggestion
    "keyboard_name": "Time RE Universal",
```

File: keyboards/foxlab/time_re/universal/readme.md [hunk 26]
```diff
@@ -0,0 +1,19 @@
+# Foxlab Time 80 Reforged Universal
+
+![Time 80 Reforged](https://i.imgur.com/BXOM9sQh.jpg)
+
+Time 80 Reforged was the second design of Fox Lab back in 2017. It has also become an iconic keyboard design of Fox Lab. Time80RE kept most of the original weight design. The clock has become thinner now and has a tempered glass cover. The front height of the keyboard has greatly reduced for optimizing for comfort. The ratio between the top and bottom cases has been adjusted for a cleaner side view.
```
Comment (noroadsleft) [inline keyboards/foxlab/time_re/universal/readme.md:5]:
```suggestion
Time 80 Reforged was the second design of Fox Lab back in 2017. It has also become an iconic keyboard design of Fox Lab. Time80RE kept most of the original weight design. The clock has become thinner now and has a tempered glass cover. The front height of the keyboard has been greatly reduced for optimizing for comfort. The ratio between the top and bottom cases has been adjusted for a cleaner side view. 
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

**Comment by matrixzj** [comment]
@noroadsleft  Thanks for careful review. All has been modified.

File: keyboards/foxlab/time_re/universal/info.json [hunk 27]
```diff
@@ -0,0 +1,103 @@
+{
+    "keyboard_name": "Time Re Universal",
```
Comment (noroadsleft) [inline keyboards/foxlab/time_re/universal/info.json:2]: I missed this on the previous review.

```suggestion
    "keyboard_name": "Time RE Universal",
```

**Review by noroadsleft — CHANGES_REQUESTED** [review]

File: keyboards/foxlab/time_re/universal/info.json (hunk 27 above)
Comment (matrixzj) [reply keyboards/foxlab/time_re/universal/info.json]: @noroadsleft  modified. Thanks

**Comment by matrixzj** [comment]
What should I do after checked with @tzarc several times , but no any response from him?

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**